pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/BrokerService.java

```diff
@@ -159,6 +165,19 @@
     
     public static final String BROKER_SERVICE_CONFIGURATION_PATH = "/admin/configuration";
     private final ZooKeeperDataCache<Map<String, String>> dynamicConfigurationCache;
+    
+    private static final int FALSE = 0;
+    private static final int TRUE = 1;
+    private static final AtomicIntegerFieldUpdater<BrokerService> TOTAL_UNACKED_MESSAGES_UPDATER =
```

Comment:
The field updater is only useful to avoid one extra object allocation. In this case, there would be 1 single instance per broker, so we can just use `AtomicInteger`